### ISSUE TITLE
Components break in hikari 2.0.0.dev110

### ISSUE THREAD

**Issue by piotr25691** [HyperGH/hikari-miru#24]
### Steps to reproduce

1. Upgrade to hikari 2.0.0.dev110
2. Send a message with any components
3. Try to use said components 
4. Observe the components failing 

### Expected result

The bot should've responded to the component interaction.

### Actual result

The bot didn't respond to any component interactions.

### System information

```shell
hikari-miru - package information
----------------------------------
Miru version: 1.1.1
Install path: /root/venv/lib/python3.10/site-packages/miru
Hikari version: 2.0.0.dev110
Install path: /root/venv/lib/python3.10/site-packages/hikari
Python: CPython 3.10.5 (GCC 12.1.0)
System: Linux x86_64 (solar) - 5.18.16-arch1-1
```

### Checklist

- [X] I have made sure to remove ANY sensitive information (bot token, passwords, credentials, personal details, etc.).
- [X] I have searched the issue tracker and have made sure it's not a duplicate. If it is a follow up of another issue, I have specified it.

**Comment by davfsa** [comment]
Well, judging from the system information you provided that states that you are using hikari 2.0.0.dev109, I am kinda confused now.

Could you please check the console for errors and attach them if possible with some minimal code to reproduce the issue?

**Comment by piotr25691** [comment]
> Well, judging from the system information you provided that states that you are using hikari 2.0.0.dev109, I am kinda confused now.
> 
> Could you please check the console for errors and attach them if possible with some minimal code to reproduce the issue?

That's an oversight on my end, I have downgraded to 109 before I wrote this bug report. I couldn't see any errors, but I can attach a reproducible code soon.

**Comment by HyperGH** [comment]
> > Well, judging from the system information you provided that states that you are using hikari 2.0.0.dev109, I am kinda confused now.
> > Could you please check the console for errors and attach them if possible with some minimal code to reproduce the issue?
> 
> That's an oversight on my end, I have downgraded to 109 before I wrote this bug report. I couldn't see any errors, but I can attach a reproducible code soon.

I'd definitiely appreciate that, along with a traceback of what the issue is.

**Comment by HyperGH** [comment]
Hey!

I managed to potentially narrow down the issue with another user on Discord, I released `v1.1.2` on PyPi, can you confirm this fixes your issue?

**Comment by piotr25691** [comment]
> Hey!
> 
> I managed to potentially narrow down the issue with another user on Discord, I released `v1.1.2` on PyPi, can you confirm this fixes your issue?

It seems to be working now. Thanks!

**Comment by HyperGH** [comment]
Closing as fixed.
